### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by damtur**
Adding a new - easier way of running tests for fargo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,20 @@ c.GetApps() // returns a map[String]fargo.Application
 # Testing
 
 Tests can be executed using docker container. See the below section to build and start 
-all the required containers. Once fargo container is running:
+all the required containers. Once eureka containers are running, and fargo container is built.
```
Comment (seh) [inline README.md]: - [ ] Capitalize "Eureka" as a proper noun.
- [ ] s/Once Eureka/Once the Eureka/
- [ ] s/container is built./image is built,/  
  For the last one, did you mean to say that once the containers are running and the _fargo_ image is built, _then_ you can run the command that follows?

File: README.md [hunk 2]
```diff
@@ -11,16 +11,20 @@ c.GetApps() // returns a map[String]fargo.Application
...
 
-1. attach shell `docker attach fargo`
-2. Checkout your test branch
-3. execute `go test -v ./...`
+Run:
+```
+docker run --rm -v "$PWD":/go/src/github.com/hudl/fargo -w /go/src/github.com/hudl/fargo hudloss/fargo:master go test -v ./...
+```
+Note: If you are running bash for windows add `MSYS_NO_PATHCONV=1 ` at the beggining
```
Comment (seh) [inline README.md]: - [ ] Capitalize "Windows" as a proper noun.
- [ ] s/beggining/beginning./

File: README.md [hunk 3]
```diff
@@ -11,16 +11,20 @@ c.GetApps() // returns a map[String]fargo.Application
...
 
 The tests may need to be run a couple of times to allow changes to propagate
 between the two eureka servers. If the tests are failing, try running them again
 approximately 30 seconds later.
 
+If you are adding new packages to godep you may want to update hudloss/fargo image first.
```
Comment (seh) [inline README.md]: - [ ] s/update hudloss/update the hudloss/

Comment (damtur) [inline README.md]: Thanks, all updated.

**Comment by damtur** [comment]
Well docker and testing is working. Right now we have a bigger problem - tests are failing on master. I'm trying to figure out why, but since I'm not very familiar with whole code it might take a while.

**Pull request merged**